Tests/test_pairwise_aligner.py

```diff
@@ -275,18 +296,22 @@ def test_smithwaterman(self):
   query_right_open_gap_score: -0.100000
   query_right_extend_gap_score: -0.100000
   mode: local
-""")
+""",
+        )
         score = aligner.score("AwBw", "zABz")
         self.assertAlmostEqual(score, 1.9)
         alignments = aligner.align("AwBw", "zABz")
         self.assertEqual(len(alignments), 1)
         alignment = alignments[0]
         self.assertAlmostEqual(alignment.score, 1.9)
-        self.assertEqual(str(alignment), """\
+        self.assertEqual(
+            str(alignment),
+            """\
  AwBw
  |-| 
 zA-Bz
-""")  # noqa: W291
+""",
+        )  # noqa: W291
```

Comment:
You need to move the ``# noqa: W291`` comment to the closing triple quote - otherwise flake8 complains about the trailing whitespace in the expected output. Same applies below.

Reply:
Understood and changed.